Dockerfile.ui-builder

```diff
@@ -0,0 +1,28 @@
+ARG image_tag=latest
+FROM elifesciences/pattern-library_assets:${image_tag} AS assets
+FROM elifesciences/php_cli:22434ef5bda09326d4c9347de7d8c2f1610a0b83
+
+USER elife
+ENV PROJECT_FOLDER=/srv/pattern-library-ui
+RUN mkdir ${PROJECT_FOLDER} && \
+    mkdir ${PROJECT_FOLDER}/public
```

Comment:
I think this is redundant as `WORKDIR` and `COPY` both create directories if they don't exist.

Reply:
They do, but with `root` ownership so `composer` fails with `/srv/pattern-library-ui/vendor does not exist and could not be created`

Reply:
Ah ok, other option is https://github.com/elifesciences/annotations/blob/a6607db87c4a9fd040b13149ebe4f1a8b3d5f4a7/Dockerfile.ci#L14.